src/vue/modules/coinpayments-deposit/components/coinpayments-form.vue

```diff
@@ -88,20 +94,32 @@ export default {
         amount: '',
       },
       isFailed: false,
-      depositDetails: null,
+      depositDetails: {
+        address: '',
+        endTime: -1,
+        amount: '',
+        payload: '',
+      },
     }
   },
   methods: {
     async submit () {
       if (!this.isFormValid()) return
       this.disableForm()
       try {
-        this.depositDetails = await this.loadDeposit(
+        let response = await this.loadDeposit(
           {
             amount: this.form.amount,
             receiver: this.balanceId,
           }
         )
+        this.depositDetails.amount = response.amount || response.data.amount
+        this.depositDetails.address = response.address || response.data.amount
+        this.depositDetails.timeout = response.timeout || response.data.timeout
```

Comment:
typo? should not `endTime` be here?

Reply:
`endTime` calculate after below

Reply:
oh, sorry. in this case better to present `timeout` in `data.depositDetails`